pybamm/solvers/scikits_ode_solver.py

```diff
@@ -72,6 +68,8 @@ def integrate(
             A function that takes in t and y and returns the Jacobian. If
             None, the solver will approximate the Jacobian.
             (see `SUNDIALS docs. <https://computation.llnl.gov/projects/sundials>`).
+        linsolver : method, optional
```

Comment:
Type should be "string"

Should there be an option in the base ODE solver class to set `linsolver`? Or does it depend on `self.method`?

Reply:
Yes, should be a string, I'll change that.

Yes, at the moment a user can't set `linsolver`. I'll provide an interface for that.